Jenkinsfile

```diff
@@ -75,14 +84,25 @@ pipeline {
                 }
             }
         }
-        stage('Publish') {
+        stage('Publish to Artifactory') {
+            environment {
+                HELM_REGISTRY="corda-os-docker-${env.BRANCH_NAME == 'main' ? 'stable' : 'dev'}.software.r3.com"
+                HELM_REPOSITORY='helm-charts'
+            }
             steps {
-                withCredentials([usernamePassword(credentialsId: 'artifactory-credentials', passwordVariable: 'PASSWORD', usernameVariable: 'USER')]) {
-                    sh '''
-                        echo $PASSWORD | helm registry login corda-os-docker-dev.software.r3.com -u $USER --password-stdin
-                        helm push corda-dev-$CHART_VERSION.tgz oci://corda-os-docker-dev.software.r3.com/helm-charts
-                    '''
-                }
+                publishHelmChart('artifactory-credentials')
+            }
+        }
+        stage('Publish to Docker Hub') {
+            when {
+                branch 'main'
```

Comment:
@davidcurrie Not close to this repo so just want to confirm.

You want every execution of the main branch to push to docker hub? rather than say just tags?

So the release of prereqs is not tied in any way to the cadence of the corda5 release itself and can happen totally independently?

Reply:
Yes. This is for Corda development and so we need to publish whatever is currently needed for developing on the Corda release currently under development. We do need to do a better job of versioning updates though. I might, for example, need an older version of this chart for developing fixes on an older Corda release. That said, at least until this update, the chart has been pretty stable.

Reply:
Ok makes sense, thanks for info  :+1: